stress-tests/data-structures/LineContainer.cpp

```diff
@@ -6,38 +6,38 @@ namespace other {
 // source: https://github.com/niklasb/contest-algos/blob/master/convex_hull/dynamic.cpp
 const ll is_query = -(1LL<<62);
 struct Line {
-    ll m, b;
-    mutable function<const Line*()> succ;
-    bool operator<(const Line& rhs) const {
-        if (rhs.b != is_query) return m < rhs.m;
-        const Line* s = succ();
-        if (!s) return 0;
-        ll x = rhs.m;
-        return b - s->b < (s->m - m) * x;
-    }
+		ll m, b;
```

Comment:
This and some other places got indented with two tabs instead of just one, somehow.

Reply:
Ok, I will check and fix